resource/addr.go

```diff
@@ -51,7 +53,7 @@ func (a *Addr) GetAddress() string {
 }
 
 func (a *Addr) Validate(sys *system.System) []TestResult {
-	ctx := context.WithValue(context.Background(), "id", a.ID())
+	ctx := context.WithValue(context.Background(), idKey{}, a.ID())
```

Comment:
The issue was:

```
resource/addr.go:54:49: SA1029: should not use built-in type string as key for value; define your own type to avoid collisions (staticcheck)
	ctx := context.WithValue(context.Background(), "id", a.ID())
```